### ISSUE TITLE
Error in gsub("</", "\\u003c/", payload, fixed = TRUE) : input string 1 is invalid UTF-8

### ISSUE THREAD

**Issue by ries9112** [vegawidget/vlbuildr#54]
Hello! I really enjoyed your talk at RStudio::conf and I was trying out your package, but ran into the error: Error in gsub("</", "\\u003c/", payload, fixed = TRUE) : input string 1 is invalid UTF-8

When using vl_chart() using the first example of the readme, I get this error, here it is with the traceback included:
![image](https://user-images.githubusercontent.com/32306488/73564845-b6cf4a00-441d-11ea-9944-6f0d2adf1c29.png)

I tried to provide a reprex using code from another talk I really enjoyed regarding using "Pins", but I have no idea if this will work for you and I am at the airport about to board a plane so I won't be responsive either, but hopefully it works?

Below is the code for a reprex:

```
library(pins)
library(vlbuildr)
#Get data from pin
#Register with board
board_register_github(name = "github", repo = "ries9112/PinsTest", branch = "master")
#Pull data from board
cryptoData <- pin_get('cryptoData','github')

#Make chart
vl_chart() %>%
   vl_add_data(values = cryptoData) %>%
   vl_mark_point() %>%
   vl_encode_x("PriceUSD") %>%
   vl_encode_y("PercChange24hVsUSD") 


# As far as I can tell I'm using cryptoData as a dataframe with two numeric variables, 
# I don't really understand the error I am getting

```

**Comment by AliciaSchep** [comment]
Thanks for trying out the package!  
I am not able to reproduce your error, although it does fail to render using that code as-is. It seems like your columns of interest are stored as character vectors even though they are quantitative -- either they need to be converted to numeric type or you can tell Vega-Lite the type:

```r
vl_chart() %>%
    vl_add_data(values = cryptoData ) %>%
    vl_mark_point() %>%
    vl_encode_x("PriceUSD:Q") %>%
    vl_encode_y("PercChange24hVsUSD:Q")
```
With setting the type the plot renders on my end. 

This also works for me:
```r
cryptoData %>% 
     transmute(PriceUSD = as.numeric(PriceUSD), PercChange24hVsUSD = as.numeric(PercChange24hVsUSD)) %>% 
    vl_chart() %>%
     vl_mark_point() %>%
     vl_encode_x("PriceUSD") %>%
     vl_encode_y("PercChange24hVsUSD") 
```
and is probably a better approach because it'll pass less data to the chart.  

Can you try the latter approach?  It might also help if there is a character encoding issue

**Comment by ries9112** [comment]
Thank you for **making** the package! Also thanks for the quick response here, I am taking off on a plane as I type this, so I can't try it out right this second, but I'll give that a go.

In terms of what I did to the data types there, I just wrote the data to a .csv and read it in using vroom::vroom() to convert all data types at once, what are your thoughts around why that approach wouldn't work? Within my session I am seeing those as numeric, but I can provide some more info there and test it more once I am home tonight or tomorrow.
